### ISSUE TITLE
Drag & Drop Merge

### ISSUE THREAD

**Issue by BrianPittVP** [microsoft/vscode#143028]
Issue Type: <b>Feature Request</b>

As Windows Users, we're all familiar with the concept of dragging and dropping folders on top of existing folders with the same name.  Windows handles this by merging the folder contents together and prompting you about overwriting any files with the same names in the same locations.  VS Code has turned this upside down and simply overwrites the entire directory with what you're copying.  This is NOT normal behavior and is a serious problem when you don't know it does this.  I have lost a bunch of files that are not in source control because of this flaw.  Please fix this!  You are a Microsoft product running in a Microsoft OS, you shouldn't be changing the rules on us!!

VS Code version: Code 1.64.2 (f80445acd5a3dadef24aa209168452a3d97cc326, 2022-02-09T22:02:28.252Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No
Remote OS version: Linux x64 4.4.0-1128-aws

**Comment by JacksonKearl** [comment]
> This actually made me lose months of progress in a project after I decided to change its structure.

All deleted files should go to your system recycling bin folder.

**Comment by 80avin** [comment]
> > This actually made me lose months of progress in a project after I decided to change its structure.
> 
> All deleted files should go to your system recycling bin folder. If they do not that's a bug.

No, they aren't in recycle bin. and, replacing the files/folders this way doesn't put them in recycle bin.

**Comment by JacksonKearl** [comment]
@80avin the files should either be in recycle bin, or traced under SCM. What happens if you outright delete a file in that workspace?

**Comment by 80avin** [comment]
Deleting the file explicitly puts it into recycle bin, unless I use shift+delete, which permanently deletes it.

and, SCM tracks them, but I hadn't committed any changes, which I agree is my fault.
